content/en/llm_observability/setup/sdk/nodejs.md

```diff
@@ -0,0 +1,713 @@
+---
+title: LLM Observability Node.js SDK Reference
+code_lang: nodejs
+type: multi-code-lang
+code_lang_weight: 30
+aliases:
+    - /tracing/llm_observability/sdk/nodejs
+    - /llm_observability/sdk/nodejs
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">LLM Observability is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+## Overview
+
+The LLM Observability SDK for Node.js enhances the observability of your Javascript-based LLM applications. The SDK supports Node.js versions 16 and newer. For information about LLM Observability's integration support, see [Auto Instrumentation][13].
+
+You can install and configure tracing of various operations such as workflows, tasks, and API calls with wrapped functions or traced blocks. You can also annotate these traces with metadata for deeper insights into the performance and behavior of your applications, supporting multiple LLM services or models from the same environment.
+
+## Setup
+
+### Prerequisites
+
+1. The latest `dd-trace` package must be installed:
+
+{{< code-block lang="shell">}}
+npm install git+https://git@github.com/DataDog/dd-trace-js.git#sabrenner/llmobs-sdk
```

Comment:
Are we going to be making users install off a branch? Or are we going to merge the SDK before we change/publish this?

Reply:
This is just here now for initial use from design partners. When I come back to give this PR a final pass once the SDK is released, I'll change this to
```
npm install dd-trace
```